Van/veb/bash/combine_files/combinefile

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+rm -rf comb_folder
+a=$(find -name 'file*')
+mkdir comb_folder
+cp ./file_math ./comb_folder
+cp ./file_nomath ./comb_folder
```

Comment:
Files from the $a should be moved into the corresponding folders
Will be merged with this incomplete version